### DISCUSSION TITLE
Custom Token Exchange Provider not invoking

### DISCUSSION THREAD

**Discussion by krishnakumarakurathi** [keycloak/keycloak#33689] [Q&A]
Hi team, I need your support on the below 

As part of OAuth2.0 Token Exchange API flow, I’ve implemented a custom token exchange provider by implementing **TokenExchangeProvider**. Also, enabled my custom provider and disabled “default“ provider with the below commands. However, for some reason my custom provider is not getting triggered. It would be helpful if someone suggests or guide me in resolving this issue.

kc.bat start --spi-token-exchange-provider-<my-custom-tokenexchange-provider-id>-enabled=true --spi-token-exchange-provider-default-enabled=false

**Comment by codespearhead** [comment]
Please confirm that this issue is still reproducible in the latest stable release of Keycloak, which currently is version `26.0.0`. If it is, provide a minimal reproducible example of your TokenExchangeProvider file.

Also, mind the extra dash you're passing to the first flag.
